Makefile

```diff
@@ -43,6 +43,18 @@ test-unit-ci:
 
 # End to end testing
 test-e2e:
-	./frontend/node_modules/nightwatch/bin/nightwatch -c tests-e2e/browserstack.conf.js -e chrome
+	# Build front-end files
+	# Set CI to a blank string to disable warnings on CI.
+	# See https://github.com/facebook/create-react-app/issues/2453
+	cd frontend && CI="" npm run build
+
+	# Run Django server and detach it
+	nohup ./manage.py runserver &
+
+	# Run end-to-end tests
+	cd frontend && node tests-e2e/local.runner.js -c tests-e2e/browserstack.conf.js -e chrome
+
+	# Kill Django server (listening on port 8000)
+	lsof -ti :8000 | xargs --no-run-if-empty kill -9
```

Comment:
ok mais un peu violent et fragile. Ca pourrait arriver qu'on lance le service sur un autre port, ou qu'un service d'un autre projet tourne déjà sur ce port etc. Un peu casse cou de se baser uniquement sur le port pour faire un kill -9.

Exemple un peu moins casse cou vu sur https://stackoverflow.com/questions/17385794/how-to-get-the-process-id-to-kill-a-nohup-process

```
nohup ./manage.py runserver &
echo $! > e2e_server_pid.txt
kill -9 `cat e2e_server_pid.txt`
rm save_pid.txt
```

Reply:
C'était la première piste que j'avais suivie, puis je suis partie sur une solution plus radicale en me disant que c'était plus court. Mais, en effet, ça peut être dangereux ! Je vais donc changer :) .

Reply:
J'ai adapté la commande car elle posait des soucis dans le Makefile.